project/addons/gut/collected_script.gd

```diff
@@ -34,180 +34,180 @@ var was_run = false
 
 
 var name = '' :
-	get: return path
-	set(val):pass
```

Comment:
Fix formatting, whitespace

Reply:
Fixed. I kept these formatting changes in a separate commit, so that it's easy to distinguish the official plugin code from our changes.